keras/utils/data_utils.py

```diff
@@ -214,7 +214,7 @@ def dl_progress(count, block_size, total_size):
             if os.path.exists(fpath):
                 os.remove(fpath)
             raise
-        progbar = None
+        progress_tracker.progbar = None
```

Comment:
whoops! Thanks for the fix. I'd lean towards yes, but I don't know the details of how locally declared python classes work between calls.